benchmark/os/uptime.js

```diff
@@ -1,14 +1,21 @@
 'use strict';
 
+const os = require('os');
 const common = require('../common.js');
-const uptime = require('os').uptime;
 const assert = require('assert');
 
+const uptime = os.uptime;
+
 const bench = common.createBenchmark(main, {
   n: [1e5],
 });
 
 function main({ n }) {
+  if (os.type() === 'OS400') {
+    console.log('Skipping: os.uptime is not implemented on IBMi');
+    process.exit(0);
+  }
```

Comment:
I think it is better to put outside `main`, on top of the file, just not to have any weird behavior.

Reply:
@mhdawson can you move this to a similar position like in other fs benchmarks?